qiskit/opflow/converters/pauli_basis_change.py

```diff
@@ -144,6 +144,7 @@ def convert(self, operator: OperatorBase) -> OperatorBase:
             cob_instr_op, _ = self.get_cob_circuit(origin_pauli)
             primitive.paulis.z = np.logical_or(primitive.paulis.x, primitive.paulis.z)
             primitive.paulis.x = False
+            primitive.paulis.phase = 0
```

Comment:
Could you add a comment here why this is necessary? Otherwise it might be hard to trace back in the future 🙂

Reply:
This is because the previous PauliTable did not have a phase and did not track it, so phase=0 was always guaranteed, but the new PauliList may change phase.